### PR TITLE
Follow up of PR #5

### PR DESCRIPTION
**Pull request by vasudevgupta7**
This is the follow up of PR #5.

Major Change:
* Text is added to the [notebook](https://colab.research.google.com/github/vasudevgupta7/gsoc-wav2vec2/blob/export-v2/notebooks/wav2vec2_saved_model_finetuning.ipynb) (while code is kept untouched)

@sayakpaul @MorganR

### CODE REVIEW

**Comment by vasudevgupta7** [comment]
I have submitted PR for exporting saved-model here: https://github.com/tensorflow/tfhub.dev/pull/65

**Review by sayakpaul — CHANGES_REQUESTED** [review]
@vasudevgupta7 great work with the notebook. Just scan it through any pesky grammatical or spelling-related bugs. For example, I caught one here (**_repositary_**):

> Finally, we have reached an end to this notebook. But it's not an end of learning TensorFlow for speech-related tasks, this repositary contains some more amazing tutorials. Feel free to go through them. In case you encounter any bug, please create an issue here.

You might want to also give [this repository](https://github.com/tulasiram58827/TTS_TFLite) a shoutout as it shows many non-trivial and relevant TFLite workflows. It's entirely up to you. 

Here are some observations: 

* The number of trainable parameters seems off. Should it be just 2? Note that if it's a Keras model you call call `count_params()` on it. `trainable_variables` returns a list and in your code length of those lists is being accounted for and NOT the actual elements that are present inside them. 
* Is it possible to operate on a small portion of the dataset and actually show how to preprocess it? It's not mandatory in this case to use TFRecords just so you know. You might also consider outputting a few samples from the training set so that the readers are aware of what they are dealing with. 
* Consider noting about the pre-trained model for at least two sentences to make things more complete. 
* See if we can also add an evaluation metric to report on the datasets. 
* Let's wrap this model as a subclass of `tf.keras.Model` as well. 

Let me know if anything is unclear.

**Comment by vasudevgupta7** [comment]
@sayakpaul, @MorganR, 

[notebook](https://github.com/vasudevgupta7/gsoc-wav2vec2/blob/export-v2/notebooks/wav2vec2_saved_model_finetuning.ipynb) is ready for final review !!

Major changes:
* worked on all the suggestions in notebook by @sayakpaul 
* `hub.KerasLayer` now can take the link directly (no need of extra thinking about signature)
* Added `argparse` support for conversion script

**Comment by vasudevgupta7** [comment]
Merging this PR to be able to access link of the notebook from TFHub. But open for any kind of feedback (will work on that in another PR).
